deployments/cert_manager.go

```diff
@@ -207,7 +208,76 @@ func (cm CertManager) apply(ctx context.Context, c *kubernetes.Cluster, ui *term
 		}, duration.ToDeployment(), duration.PollInterval())
 	if err != nil {
 		if strings.Contains(err.Error(), "Timed out after") {
-			return errors.Wrapf(err, "failed to create clusterissuer selfsigned-issuer")
+			return errors.Wrapf(err, "failed to create cluster-issuer %s", SelfSignedIssuer)
+		}
+		return err
+	}
+
+	// With the self signed issuer in place it is now possible to bootstrap
+	// Epinio's private CA. Phase 1, the CA root certificate, signed by self
+	// signed.
+
+	caCert := fmt.Sprintf(`{
+		"apiVersion" : "cert-manager.io/v1alpha2",
+		"kind"       : "Certificate",
+		"metadata"   : {
+			"name"      : "epinio-ca",
+			"namespace" : "%s"
```

Comment:
nit: this is not needed when using a namespaced client, like in 256

Reply:
Will fix.

Reply:
Done, commit 0c9dd5aea7099f744bfc8571d32ec374e7e243c6